File: Data Structures/StudentSort/src/Student.java

public class Student {
	
	private int rollNo;
	private String studentName;
	
	public Student(String n, int rn )
	{
		rollNo = rn;
		studentName = n;
	}
	
	public String toString() {
		String str = "";
		str = "Roll: " + rollNo + ", Name: " + studentName;
		return str;
	}
	
	public void setRollNo(int rollNo) {
		this.rollNo = rollNo;
	}

	public int getRollNo()
	{
		return rollNo;
	}

}

File: MS.Net/18feb/Solution18feb/IndexerClassLibrary1/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IndexerClassLibrary1
{
    public class Student
    {
       
        
        public int RollNo { get; set; } 
        public string Name { get; set; }
        
        private int[] Marks = new int[5];

        public Student(int rollNo, string name, int[] marks)
        {
            RollNo = rollNo;
            Name = name;
            Marks = marks;
        }
        public Student(int rollNo, string name)
        {
            RollNo = rollNo;
            Name = name;
        }
        public int this[int index]
        {
            get { return Marks[index]; }
            set {Marks[index] = value; }
        }
    public override string  ToString()
        {
            return "Roll No: " + RollNo +", Student Name: "+Name;
        }
    }
}
